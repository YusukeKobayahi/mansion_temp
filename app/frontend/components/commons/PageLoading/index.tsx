import styles from "./index.module.scss";

const PageLoading: React.FC = () => {
  return (
    <div className={styles.skCircleFade}>
      <div className={styles.skCircleFadeDot}></div>
      <div className={styles.skCircleFadeDot}></div>
      <div className={styles.skCircleFadeDot}></div>
      <div className={styles.skCircleFadeDot}></div>
      <div className={styles.skCircleFadeDot}></div>
      <div className={styles.skCircleFadeDot}></div>
      <div className={styles.skCircleFadeDot}></div>
      <div className={styles.skCircleFadeDot}></div>
      <div className={styles.skCircleFadeDot}></div>
      <div className={styles.skCircleFadeDot}></div>
      <div className={styles.skCircleFadeDot}></div>
      <div className={styles.skCircleFadeDot}></div>
    </div>
  );
};

export default PageLoading;
