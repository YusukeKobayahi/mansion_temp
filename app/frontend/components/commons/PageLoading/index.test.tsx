import { render } from "@testing-library/react";
import PageLoading from "../PageLoading";

describe("components/commons/PageLoading", () => {
  it("正常に表示される", () => {
    const { asFragment } = render(<PageLoading />);
    expect(asFragment()).toMatchSnapshot();
  });
});
