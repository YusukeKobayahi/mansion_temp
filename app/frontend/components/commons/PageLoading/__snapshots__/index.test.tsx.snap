// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/commons/PageLoading 正常に表示される 1`] = `
<DocumentFragment>
  <div
    class="skCircleFade"
  >
    <div
      class="skCircleFadeDot"
    />
    <div
      class="skCircleFadeDot"
    />
    <div
      class="skCircleFadeDot"
    />
    <div
      class="skCircleFadeDot"
    />
    <div
      class="skCircleFadeDot"
    />
    <div
      class="skCircleFadeDot"
    />
    <div
      class="skCircleFadeDot"
    />
    <div
      class="skCircleFadeDot"
    />
    <div
      class="skCircleFadeDot"
    />
    <div
      class="skCircleFadeDot"
    />
    <div
      class="skCircleFadeDot"
    />
    <div
      class="skCircleFadeDot"
    />
  </div>
</DocumentFragment>
`;
